quartz/i18n/index.ts

```diff
@@ -1,44 +1,46 @@
 import { Translation, CalloutTranslation } from "./locales/definition"
+import ar from "./locales/ar-SA"
```

Comment:
Let's keep these change to a minimum. Let's just add the `import` after `uk`

Reply:
done)